source/material_model/interface.cc

```diff
@@ -236,7 +236,7 @@ namespace aspect
     }
 
 
-
+    DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
```

Comment:
Otherwise we get a warning that we use cell, which is deprecated (and I can not put the block just around the single line, because it's in the constructor). 
Or did you mean I should add a comment why this is necessary?